packages/bierzo-wallet/src/components/dummy/DummyRoot.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
If those components are not used, please, remove them.

Reply:
Fixed by [14f6201](https://github.com/iov-one/ponferrada/pull/79/commits/14f6201854bd623d3fb74608509f7e5b7c587dc8)